_errata/errata-print02-ch02-lst03-superfluous-punctuation

```diff
@@ -0,0 +1,30 @@
+---
+printing: second
+chapter: 1
+page: 27
+kind: code
+reporter: Kyle Strand
+date: 2023-08-27
+---
+
+Listing 2-3 reads:
+
+```rust
+impl String {
+    pub fn contains(&self, p: &dyn Pattern) -> bool {
+        p.is_contained_in(&*self)
+    }
+}
+```
+
+The `&*` before `self` has no effect and should be removed. The listing would then read:
+
+```rust
+impl String {
+    pub fn contains(&self, p: &dyn Pattern) -> bool {
+        p.is_contained_in(self)
+    }
+}
+```
+
+Resolves #39
```

Comment:
This should just be in the Git commit message, and doesn't need to be in the Markdown file.